### ISSUE TITLE
Contact Form: PHP notice -- array_keys() expects parameter 1 to be array

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#2991]
```
"PHP Warning:  array_keys() expects parameter 1 to be array, string given in /xxx/xxx/xxx/wp-content/plugins/jetpack/modules/contact-form/grunion-contact-form.php on line 1175"
```
https://github.com/Automattic/jetpack/blob/3.7.2/modules/contact-form/grunion-contact-form.php#L1175

Reported in [#2610399-org](https://wordpress.org/support/topic/array_keys-expects-parameter-1-to-be-array?replies=1&view=all)

cc @RavanH

**Comment by richardmtl** [comment]
another report in 2426526-t

**Comment by bisko** [comment]
The proposed fix should fix the error from appearing in the future.

**Comment by jeherve** [comment]
Also reported here:
https://wordpress.org/support/topic/problem-with-jetpack-module-with-contact-form?replies=1&view=all

**Comment by jeherve** [comment]
Fixed in #3222
